### ISSUE TITLE
Support for changing between left/right position of window decorations

### ISSUE THREAD

**Issue by Jacalz** [fyne-io/fynedesk#140]
It would be great if it would be possible to change window decorations such as close button and maximize buttons to be on either the left or right side of windows. Having them on the left is probably great for people coming from MacOS, but users familiar with having them on the right will most certainly struggle a bit (I do).

**Comment by AlbinoGeek** [comment]
If this is a wishlist, I'd love to be able to configure the decorator like you can on Gnome:

```
gsettings set org.gnome.desktop.wm.preferences button-layout "close,minimize,maximize:"
```

![img1](https://i.stack.imgur.com/bX5OV.png)

```
gsettings set org.gnome.desktop.wm.preferences button-layout ":minimize,maximize,close"
```

![img2](https://i.stack.imgur.com/KZvbu.png)

```
gsettings set org.gnome.desktop.wm.preferences button-layout "close:minimize,maximize"
```

![img3](https://i.stack.imgur.com/NXoey.png)

**Comment by Jacalz** [comment]
Let’s stick to left and right placement for now. I don’t think we want to go down the KDE route if being able to change everything.

**Comment by andydotxyz** [comment]
This was released in 0.2
